documentsManager/templates/home.html

```diff
@@ -197,6 +190,7 @@ <h1 class="eds-empty-state__title eds-text--center eds-text-hs eds-text-color--g
 	        </div>
 	    </div>
 	</div>
+	<input type="hidden" id="eb_user" value="{{is_eb_user}}">
```

Comment:
```suggestion
	<input type="hidden" id="eb_user" value="{{is_eb_user}}">
```
Pls add some spaces before and after the variable {{ is_eb_user }}

Reply:
Ok.